README.md

```diff
@@ -25,6 +25,30 @@ Simply install caustics from PyPI:
 pip install caustics
 ```
 
+## Minimal Example
+
+```python
+import matplotlib.pyplot as plt
+import caustics
+
+C = caustics.cosmology.FlatLambdaCDM()
+
```

Comment:
I perhaps take issue with this basic example. It's the wrong way to use caustic even if in terms of code it is perhaps the simplest. 

A user copying and implementing this code in their pipeline would find caustics slow and clunky

Reply:
I agree, I'll change it to be more in line with proper style